### PR TITLE
XWIKI-22205: Underline inline link in CSRF confirmation Panel

### PR DESCRIPTION
**Pull request by Sereza7**
# Jira URL

https://jira.xwiki.org/browse/XWIKI-22205

# Changes

## Description

* Fixed all the translation values by adding a class to force this specific link to be underlined.

## Clarifications

* This translation breaks a lot of our translation rules, but reworking it (and probably deprecating the old translation key) is out of the scope of this PR. I decided to update all the instances of this translation instead. It was possible to figure out what markup needs an update whatever the language, so this solution seemed correct.

# Screenshots & Video

None. 

# Executed Tests

None. This is a style only change with a very low scope.

# Expected merging strategy

* Prefers squash: Yes 
* Backport on branches:
  * None

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_bg.properties [hunk 1]
```diff
@@ -3576,7 +3576,7 @@ platform.index.attachments.emptyvalue=
...
+### Missing: csrf.confirmation=<p>This request contains an invalid authentication information.</p><p>This might happen in the following situations:</p><ul><li>You left the editor open in another window/tab and logged off and on again</li><li>Your authentication token expired after a long period of inactivity</li><li>Somebody tried to perform a CSRF attack</li></ul><p class="force-underline">If you are sure that none of these situations apply in your case, you might have found a bug. We are sorry about that, please report it on <a href="http://jira.xwiki.org/">XWiki JIRA</a></p><p>Do you want to resend the request? If unsure, say <strong>No</strong>.</p>
```
Comment (surli) [inline xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_bg.properties:3579]: You don't need to edit other translations file and you actually shouldn't: those are commented so they aren't used.

**Review by surli — COMMENTED** [review]
I'd really won't go towards this solution. The translation is already bad and adding the class is really not improving anything. I prefer we keep having the bug and we fix once for all the translations to get rid of all html.

File: xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties [hunk 2]
```diff
@@ -3640,7 +3640,7 @@ XWiki.SearchSuggestSourceClass_activated=Activated
...
-csrf.confirmation=<p>This request contains an invalid authentication information.</p><p>This might happen in the following situations:</p><ul><li>You left the editor open in another window/tab and logged off and on again</li><li>Your authentication token expired after a long period of inactivity</li><li>Somebody tried to perform a CSRF attack</li></ul><p>If you are sure that none of these situations apply in your case, you might have found a bug. We are sorry about that, please report it on <a href="http://jira.xwiki.org/">XWiki JIRA</a></p><p>Do you want to resend the request? If unsure, say <strong>No</strong>.</p>
+csrf.confirmation={0}This request contains an invalid authentication information.{1}{2}This might happen in the following situations:{3}{4}You left the editor open in another window/tab and logged off and on again{5}{6}Your authentication token expired after a long period of inactivity{7}{8}Somebody tried to perform a CSRF attack{9}{10}If you are sure that none of these situations apply in your case, you might have found a bug. We are sorry about that, please report it on {11}XWiki JIRA{12}{13}Do you want to resend the request? If unsure, say {14}No{15}.{16}
```
[the remark this answers is not in the document]
Comment (surli) [reply xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties:3643]: > Should I deprecate this exact key and introduce a new one with the correct amount of parameters?

Yes you should do that.

[the remark this answers is not in the document]
Comment (surli) [reply xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties:3643]: Note that we could also decide to split this translation in multiple keys: e.g. one key for the message title, one for a description, one for details. Might be more clean.

**Comment by Sereza7** [comment]
Here is what the UI looks like after the changes proposed in this PR:
Screenshot from 2024-11-04 16-14-44
Screenshot from 2024-11-04 15-29-42

**Pull request merged**